front-end/src/screens/Settings/email.tsx

```diff
@@ -1,21 +1,25 @@
 import React, { useState, useContext, useEffect } from 'react'
 
+import { Icon } from 'semantic-ui-react'
```

Comment:
Please put external deps together separated by a blank line from the internal deps.
so effecticely, this one should be bellow `react`